### ISSUE TITLE
fix LZO compression

### ISSUE THREAD

**Issue by Pierrotws** [ryba-io/ryba#46]
LZO seems to not work properly, at least in hbase

```
hbase> create 'toto', {NAME => 'name', COMPRESSION => 'LZO', BLOOMFILTER => 'ROW'} 
ERROR: org.apache.hadoop.hbase.DoNotRetryIOException: Compression algorithm 'lzo' previously failed test
```

**Comment by IDerr** [comment]
When I tried this, on ryba-env-metal I have this error 

```
hbase(main):001:0> create 'toto', {NAME => 'name', COMPRESSION => 'LZO', BLOOMFILTER => 'ROW'}

ERROR: org.apache.hadoop.hbase.DoNotRetryIOException: java.lang.RuntimeException: java.lang.ClassNotFoundException: com.hadoop.compression.lzo.LzoCodec Set hbase.table.sanity.checks to false at conf or table descriptor if you want to bypass sanity checks
```

**Comment by wdavidw** [comment]
is the package installed ? there used to be some incompatibilities in centos6 between epel and hdp-utils if i remember

**Comment by IDerr** [comment]
Hmm, I tried on centos7, I will reinstall a cluster on centos6 to try

**Comment by wdavidw** [comment]
no need to reinstall centos6, i'm not even sure we really tries lzo on it, is the package correctly installed on centos7 ?

**Comment by IDerr** [comment]
I had no problem with lzo, even on fresh installs.

**Comment by IDerr** [comment]
But @leopaul36 or @lucasbak can confirm that ?

**Comment by IDerr** [comment]
Still the same error but fixed by installing hadooplzo hadooplzo-native in hbase master

The commit is coming.

**Comment by IDerr** [comment]
Fixed by 2f74bab
